app/assets/stylesheets/notification.scss

```diff
@@ -0,0 +1,32 @@
+@import "colors";
+
+/* Styling Container e.g. Position, size, etc. */
+#toast-container{
```

Comment:
do you want rly are seperate conatiner for hte notificartions with backgroudn and shadow?